Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM golang:alpine
 WORKDIR /usr/src/app
 
 RUN adduser -u 9000 -D app
-ADD build/codeclimate-gofmt /usr/src/app
+COPY . /usr/src/app
```

Comment:
@GordonDiggs thanks good idea! Updated to only copy the binary.

The image size is still bigger than before - probably due to using the larger golang image instead of alpine*:

```
codeclimate/codeclimate-gofmt      latest     80a2cca69578        About a minute ago   247.6 MB
codeclimate/codeclimate-gofmt      newer     376f5bbe035a        8 hours ago          270.5 MB
codeclimate/codeclimate-gofmt       old       8b5b9e71bd0e        6 months ago         218.3 MB
```

Currently looking at the circle failure. Engine works locally. 

I'm suspicious of 

```
stat /var/lib/docker/btrfs/subvolumes/7c5f9c8c0809d25df2ac0fbad945c4d8724b87ae197dd269da8dfaaa0b9b8998/
```

 before /usr/src/app/codeclimate-gofmt

\* `yunspace/alpine-golang` no longer exists. There's also a `golang:alpine` image I considered, but ran into an issue missing git during build script, and also their docs suggest golang:latest might be the way to go for long lived projects: https://hub.docker.com/_/golang/